src/main/webapp/i18n/en/programmingExercise.json

```
@@ -502,6 +502,7 @@
                 "submissionLimitTitle": "Submission limit",
                 "submissionLimitDescription": "The number of submissions a participant can make before the system enforces the selected policy.",
                 "editInGradingInformation": "The submission policy can only be edited and toggled on the grading page of the programming exercise!",
+                "goToGradingToEditInformation": "Go to the Grading Page to edit Submission Policy.",
```

```suggestion
                "goToGradingToEditInformation": "Go to the grading page to edit submission policy.",
```
This is a proper sentence and not a heading